jest.config.js

```diff
@@ -9,5 +9,7 @@ module.exports = {
     '<rootDir>/src/utils/test-deprecations.tsx',
     '<rootDir>/src/utils/test-helpers.tsx'
   ],
-  testMatch: ['<rootDir>/(src|codemods)/__tests__/**/*.test.[jt]s?(x)', '!**/*.types.test.[jt]s?(x)']
+  testMatch: ['<rootDir>/(src|codemods)/__tests__/**/*.test.[jt]s?(x)', '!**/*.types.test.[jt]s?(x)'],
+  // @primer/behaviors is ESM, so needs to be transformed
```

Comment:
nit: this didn't make sense to me — it seems like you're saying something needs to be transformed, but then the code seems like you're setting it to be ignored?